packages/vsce/src/resources/actions/TransactionActions.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { IResourceAction, IResourceContext, ResourceTypes } from "@zowe/cics-for-zowe-explorer-api";
+import { ITransaction } from "../../doc";
+
+export function getTransactionActions(): IResourceAction[] {
+  return [
+    {
+      id: "CICS.CICSLocalTransaction.INQUIRE",
+      name: "Inquire Program",
+      resourceType: ResourceTypes.CICSLocalTransaction,
+      action: "cics-extension-for-zowe.inquireProgram",
+    },
+    {
+      id: "CICS.CICSLocalTransaction.ENABLE",
+      name: "Enable Transaction",
```

Comment:
```suggestion
      name: "Enable",
```

I reckon we can just put enable/disable - we know we're in a transaction. Happy to reconsider this in future if things become more confusing.

Reply:
OK - I'm happy to stay in line with the context menu. In that case we'll need to update program to add the resource name :-)